### ISSUE TITLE
Potential Case Handling Bug

### ISSUE THREAD

**Issue by sophiestrazie** [mitchelloharawild/icon#71]
Situation: I wanted to integrate a [teddy bear](https://fontawesome.com/icons/teddy-bear?s=thin&f=classic) (it doesn't have to be this one). This shouldn't be relevant because I'm not knitting yet, but just in case it is, know that I'm working in Xaringan (testing this with RMarkdown chunks). 

`fontawesome("teddy-bear", style = "regular")` produces:
> Error in `icon_fn$get()`:
> ! The `teddy-bear` icon could not be found in this icon set.
> Backtrace:
>  1. icons::fontawesome("teddy-bear", style = "regular")
>  2. icon_fn$get(c(style, name))

and when I change it to just bear and style regular I get:
>  Error in `icon_fn$get()`:
> ! The `bear` icon could not be found in this icon set.
> Backtrace:
>  1. icons::fontawesome("bear", style = "regular")
>  2. icon_fn$get(c(style, name))

the traceback for bear 
<img width="534" alt="Screenshot 2023-02-18 at 6 10 27 PM" src="https://user-images.githubusercontent.com/81820608/219903437-a4c81c3a-39cc-4d07-94f8-4e8241b16b96.png">

While `icons::icon_find('bear')` produces:

> $set
> Font Awesome icon set (version 6.3.0)
> $path
> [1] "bear"
> 
> attr(,"class")
> [1] "icon_dir" "list" 

`icons::icon_find('bear')` outputs
> `named list()

TLDR: Does the bear exist and how can I access it/versions of it?

**Comment by mitchelloharawild** [comment]
Apologies for not getting back to you sooner, for anyone who may be having a similar issue the `teddy-bear` icon is part of font awesome _pro_, which is currently not directly supported by the package. If you do have a pro license, you can download the svgs you wish to use, and then create a custom icon set using the `icons::icon_set()` function.
